### DISCUSSION TITLE
BepInEx isn't loaded into the game.

### DISCUSSION THREAD

**Discussion by kacejot** [BepInEx/BepInEx#327] [Help and Q&A] [answered]
Hi,
I just want to try to write some plugins for Valheim using GitHub release without using BepInExPack for Valheim.
I tried the latest release (5.4.17) and I followed [this instruction](https://docs.bepinex.dev/master/articles/user_guide/installation/unity_mono.html). 
And I'm stuck on the third item in the instruction - "Do the first-time run to generate configuration files".
After I run the valheim.exe nothing happens in the BepInEx folder, no config folder is created.
Could you please suggest what I'm doing wrong?

P.S. I also checked loaded binaries into the process. I just found this binary is loaded
![image](https://user-images.githubusercontent.com/20083419/139709169-a133d530-d69b-42ef-9ff3-f191e549730d.png)

**Comment by ghorsington** [comment, marked as answer]
Greetings!

Valheim needs unstripped core libraries and an appropriate UnityDoorstop configuration (see #182). BepInExPack for Valheim includes the necessary config and the files which is why the pack exists in the first place.

In order to use base BepInEx (which to be fair is the same as BepInExPack for the exception of premade configs, included unstripped corlibs and one plugin you can remove), you need to configure it as noted in comments of #182 and include necessary unstripped game assemblies.
